ui/src/app/shared/header/header.component.ts

```diff
@@ -174,7 +228,8 @@ export class HeaderComponent implements OnInit {
   }
 
   shouldDisplayStatusButtons(): boolean {
-    return !URLSearchParamsUtils.unpackURLSearchParams(
+    // jobs is only populated on multi-job views
+    return this.jobs && !URLSearchParamsUtils.unpackURLSearchParams(
```

Comment:
Predicating whether to show UI elements (both here and with the paginator) on the nullness of the jobs input is somewhat unintuitive and sort of breaks the component modularity agreement. Even though it might be redundant, consider providing a separate bool showControls input and then setting it to true for JobsList and false for JobDetails. That make this behavior a lot more transparent and increases the useability of this module.